tests/scripts/dind-cluster.sh

```diff
@@ -18,12 +18,12 @@ set -o nounset
 set -o pipefail
 set -o errtrace
 
-if [ $(uname) = Darwin ]; then
+if [ "$(uname)" = Darwin ]; then
   readlinkf(){ perl -MCwd -e 'print Cwd::abs_path shift' "$1";}
 else
   readlinkf(){ readlink -f "$1"; }
 fi
-DIND_ROOT="$(cd $(dirname "$(readlinkf "${BASH_SOURCE}")"); pwd)"
+DIND_ROOT="$(cd "$(dirname "$(readlinkf "${BASH_SOURCE[0]}")"); pwd")"
```

Comment:
Extra `"` was added after pwd, which is a bug. Also use `&&` like above.

```suggestion
DIND_ROOT="$(cd "$(dirname "$(readlinkf "${BASH_SOURCE[0]}")")" && pwd)"
```